src/str.nim

```diff
@@ -225,6 +225,16 @@ proc adjust_by(A:var tread, B:tread, opts:Options): bool =
   elif A.mapping_quality >= opts.min_mapq or A.flag.proper_pair:
     A.position += uint32(A.align_length.float / 2'f + 0.5) # Record position as middle of A
 
+# Return true if both reads in pair are STR, or one is STR, one low mapping qual
+proc unplaced_pair*(A:var tread, B:tread, opts:Options): bool =
+  if A.p_repeat > opts.proportion_repeat and B.p_repeat > opts.proportion_repeat:
```

Comment:
what about a special-case to save a pair where both reads are STR but 1 has a mapping-quality of 60? I assume that will happen infrequently so might not be an issue.

Reply:
It's possible for an STR read to have very high mapping quality but still be incorrectly mapped. This was causing a lot of false positives. What can happen is there is only one of that STR repeat unit in the genome, and it is long. However the STR that is expanded in the individual is at a different, novel locus.